src/main/java/org/opentripplanner/apis/transmodel/model/timetable/DatedServiceJourneyQuery.java

```diff
@@ -93,72 +94,39 @@ public static GraphQLFieldDefinition createQuery(
           .type(new GraphQLList(new GraphQLNonNull(Scalars.GraphQLString)))
       )
       .dataFetcher(environment -> {
-        Stream<TripOnServiceDate> stream = GqlUtil
-          .getTransitService(environment)
-          .getAllTripOnServiceDates()
-          .stream();
-
+        var authorities = mapIDsToDomainNullSafe(environment.getArgument("authorities"));
```

Comment:
See below. With my suggestion we have pushed the null-safety into the model as well and we do not need to do it here any more, but I would leave it here since we have not converted the entire API over to this improvement. But, a comment might be good:

```suggestion
        // The null safety is not needed here - taken care of by the request object, but reuse
        // mapping method and lave this improvement until all APIs is pushing this check in the 
        // domain request.
        var authorities = mapIDsToDomainNullSafe(environment.getArgument("authorities"));
```

Reply:
Adding the comment.